### PR TITLE
Update the iCub user enviroment documentation

### PR DESCRIPTION
**Pull request by GiulioRomualdi**
This closes https://github.com/icub-tech-iit/documentation/issues/324

### CODE REVIEW

**Review by pattacini — CHANGES_REQUESTED** [review]
Thank you @GiulioRomualdi!
For consistency, would you mind updating also the following snippet https://github.com/icub-tech-iit/documentation/blob/5d919960f2d71e25d845b1eb4b899ed0c389ae43/docs/icub_operating_systems/pc104/the-icub-live-details.md?plain=1#L302-L351

as reported in https://github.com/icub-tech-iit/documentation/issues/324#issuecomment-1787248578?

**Comment by GiulioRomualdi** [comment]
Hi @pattacini I updated the file in https://github.com/icub-tech-iit/documentation/pull/325/commits/3323f66a3b0e179d2374d09644c258ada9314b33

**Review by pattacini — APPROVED** [review]
Super! Thanks.

File: docs/icub_operating_systems/icubos/user-env.md [hunk 1]
```diff
@@ -42,34 +42,36 @@ the above files must be renamed to `.bashrc_iCub` and saved in the user home dir
...
-else
-  ICUBRC_FILE="/home/icub/.bashrc_iCub"
-fi
-if [ -f "$ICUBRC_FILE" ]; then
-  source $ICUBRC_FILE
+```bash
+if [[ $- == *i* ]] || [[ -n "$SSH_CLIENT" ]] || [[ -n "$SSH_TTY" ]]; then
+  #Load the ergoCub custom bashrc
+  if [ "$HOME" != "" ]; then
+      ICUBRC_FILE="${HOME}/.bashrc_ergoCub"
```
Comment (lrapetti) [inline docs/icub_operating_systems/icubos/user-env.md:49]: This was changed from `.bashrc_iCub` to `.bashrc_erogCub`, probably you missed it @pattacini @GiulioRomualdi. 

btw is there an equivalent documentation for ergoCub right?

Comment (pattacini) [reply docs/icub_operating_systems/icubos/user-env.md:49]: Thanks for pointing that out! I'll revert these changes.

>btw is there an equivalent documentation for ergoCub right?

We have the following doc, which doesn't go into the same level of details:
- https://icub-tech-iit.github.io/documentation/ergocub_operating_systems/

Comment (pattacini) [reply docs/icub_operating_systems/icubos/user-env.md:49]: Fixed via https://github.com/icub-tech-iit/documentation/commit/e8b2f556e5cdbc7ac4b854dea81161433b93b3c8.

**Pull request merged**